### ISSUE TITLE
GaussianBlur in data augmentation

### ISSUE THREAD

**Issue by mariehbourget** [neuropoly/axondeepseg#288]
GaussianBlur doesn’t have the option for nearest neighbour interpolation in the release 3.0.
As discussed:
- this may cause problems with mask having multiple classes after augmentation,
- we could do our own nearest neighbour by thresholding the results of the augmentation,
- do we really want to have the GausianBlur option for data augmentation? ( see #21 )

**Comment by jcohenadad** [comment]
> * do we really want to have the GausianBlur option for data augmentation? ( see #21 )

i don't think we do-- @mariehbourget i suggest you start a PR which purpose will be to remove GaussianBlur option. Thanks!
